.github/workflows/sonar.yml

```diff
@@ -8,7 +8,22 @@ on:
     types: [opened, synchronize, reopened, ready_for_review]
 
 jobs:
+
+  add-copyright:
+    if: github.event.pull_request.draft == false
+    uses: Modino-io/.github/.github/workflows/add-copyright.yaml@ci/712-copyright
+    with:
+      CALLER_REPOSITORY: ${{ github.repository }}
+      CALLER_REPOSITORY_VERSION: ${{ github.head_ref }}
+      RUNS_ON: "ubuntu-latest"
+      ADD_COPYRIGHT_TO: "${{ github.repository }}/scss ${{ github.repository }}/js ${{ github.repository }}/css"
```

Comment:
JS and CSS files are public and not confidential by nature; we'd need a different copyright notice for this specific case, like this basic one:

```
<%=copyright_software%> - <%=copyright_software_description%> 
(C) Copyright <%=copyright_holders.join(', ')%> <%=copyright_years.join(', ')%>. All Rights Reserved.
```

The license text mentioned at https://github.com/Modino-io/modino-devops/blob/main/copyright/modino_license.txt is to be used only with source code only. Not resulting artifacts that are deployed.